packages/wallet/backend/src/utils/helpers.ts

```diff
@@ -16,7 +16,7 @@ export const transformAmount = (
 }
 
 export const urlToPaymentPointer = (url: string) =>
-  url.split('/').slice(0, 4).join('/')
+  url.split('/incoming-payments/')[0]
```

Comment:
@raducristianpopa as in local playground in rafiki the payment pointers are `accounts/gfranklin`
when extracting the payment pointer from an incoming payment it was creating it as `accounts` only
this change should extract the full payment pointer

Reply:
I think this is what we are looking for: 
- https://github.com/interledger/rafiki/issues/1005